deploy/helm/jumpstarter/charts/jumpstarter-controller/templates/rbac/role.yaml

```diff
@@ -35,60 +29,16 @@ rules:
   verbs:
   - create
 - apiGroups:
-  - jumpstarter.dev
+  - authentication.k8s.io
   resources:
-  - clients
+  - tokenreviews
```

Comment:
We don't need tokenreviews anymore, right?, may be it's time to remove the annotation to require this in rbac.

Reply:
Done, also dropped the now unused tokenholder service account.